tsnet/tsnet.go

```diff
@@ -178,8 +188,35 @@ func (s *Server) start() error {
 		return fmt.Errorf("%v is not a directory", s.rootPath)
 	}
 
-	// TODO(bradfitz): start logtail? don't use filch, perhaps?
-	// only upload plumbed Logf?
+	cfgPath := filepath.Join(s.rootPath, "tailscaled.log.conf")
+
+	lpc, err := logpolicy.ConfigFromFile(cfgPath)
+	switch {
+	case os.IsNotExist(err):
+		lpc.Initialize(logtail.CollectionNode)
```

Comment:
In normal Go conventions, `lpc` would be nil here, because unless documented explicitly, a func returning (T, error) should be assumed to have a `nil` or otherwise undefined T when error is non-nil.

So I think you should precede this with:

   lpc = new(...)
 
But at that point I'd rather see an explicit constructor, like:

    lpc = logtail.NewFoo(logtail.CollectionNode)

Reply:
done